### ISSUE TITLE
Ubuntu Server: Turn the tables and review me!

### ISSUE THREAD

**Issue by s-makin** [canonical/open-documentation-academy#170]
## Task

Since the Ubuntu Server team joined the Open Documentation Academy last month, many of you have been submitting pull requests against our documentation, and had me reviewing and leaving feedback for you. Now it's your turn! I have a pull request adding a first draft of a new page, and I would love to get your feedback on it.

## Details

Reviewing is an important and often overlooked skill. This PR [adds an introduction](https://github.com/canonical/ubuntu-server-documentation/pull/153) to the cloud-init tool, and as a first draft it's quite bare-bones and could use some reviews to polish it up.

Some things to keep in mind when reviewing:
- **The goal of the piece**: to provide a high-level introduction to the tool (what it is, who it's for, what it's intended to help with)
- **Not the goal of the piece**: to explain how cloud-init works, or to be very detailed.
- **The audience**: The typical user of cloud-init is a system administrator or developer (so, highly technical), but since this piece will live in the Server docs, it should be more accessible and suitable to a much broader audience.

Please add your thoughts, opinions, suggestions, and comments directly against the PR. The feedback can be on any and all aspects; from the structure and organisation, to the wording of the headings and text, to any topics or points you think are missing that you would have found helpful or interesting to know about. 

Try to be specific about feedback - it's fine to be critical, but try to identify what exactly is wrong or that you don't like about it. It's also fine to disagree with each other, but please do so constructively :)

The PR will remain open for the next week, and I'll be aiming to get it polished up and merged by Friday 28th February, so if you would like to leave feedback, please do so before then! I'll start addressing comments by Wednesday so that I can get a good sense of which parts people have thoughts about.

## Useful links

- In this case, you'll probably need to also look at the [cloud-init documentation](https://docs.cloud-init.io/en/latest/)
- If you are new to the Canonical Open Documentation Academy, you may also want to review the Getting Started content.

Thanks in advance for your help!

**Comment by network-charles** [comment]
Nice!

**Comment by davidekete** [comment]
Hi @s-makin, I'd like to work on this issue.

**Comment by s-makin** [comment]
Thanks @network-charles - I already spotted your review!

Thanks @davidekete - feel free to go ahead and start reviewing :)

**Comment by s-makin** [comment]
Thanks for your helpful comments and suggestions! I hit my deadline so the PR has now been merged - closing this issue :)

**Comment by network-charles** [comment]
You are welcome
